.gitignore

```diff
@@ -23,3 +23,5 @@
 /dist/
 /nbdist/
 /.nb-gradle/
+/node_modules/
+/src/main/resources/public/dist/
```

Comment:
In dem Pfad hast du doch aber schon eine Datei eingecheckt. Bitte mal mit Sven abstimmen ob das sinnvoll ist, oder ob die Datei vor dem maven build schon da sein muss.